src/models/timeslot.db.model.ts

```diff
@@ -6,4 +6,5 @@ export interface TimeSlotDbModel {
     index: string;
     lesson?: LessonDbModel;
     room?: LocationDbModel;
+    customized?: boolean;
```

Comment:
Can you please elaborate again why do we need it? The initial idea was that if timeslot is on class belongs to class and if on student it is the exception on the class timeslot.

Reply:
Yes sure, @dimkinv.
 without this flag you can't tell the difference between the exception and the class's time slot.
As for today, you get a merged schedule from server and return only one possible exception per student.